pandas/io/excel/_xlrd.py

```diff
@@ -30,7 +30,7 @@ def __init__(
             Object to be parsed.
         {storage_options}
         """
-        err_msg = "Install xlrd >= 1.0.0 for Excel support"
+        err_msg = "Install xlrd >= 2.0.1 for Excel support"
```

Comment:
```suggestion
        err_msg = "Install xlrd >= 2.0.1 for xls Excel support"
```

Maybe to nudge that this is only for xls

Reply:
Yea - that's great, thanks.